test/smoke/client-s3.test.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const test = require('node:test')
+const assert = require('node:assert')
+const helper = require('../lib/agent_helper')
+const {
+  DESTINATIONS: { TRANS_SEGMENT }
+} = require('../../lib/config/attribute-filter')
+
+test('@aws-sdk/client-s3 functionality', async (t) => {
+  const { version, name } = require('@aws-sdk/client-s3/package')
+  // eslint-disable-next-line no-console
+  console.log(`AWS package: ${name} version: ${version}`)
```

Comment:
Why are we logging this?

Reply:
we do caret match on the dep on install and in the past we had issues and it was hard to see which version of the dep caused the issue